### ISSUE TITLE
Help section isn't keyboard operable

### ISSUE THREAD

**Issue by bmbc-dev-detec** [LocalGovIMS/localgov-ims#332]
The help section toggles cannot be opened using just a keyboard. Also when testing with VoiceOver screen reader there's no indication that the questions are interactive or toggable.

When content can be operated through a keyboard or alternate keyboard, it is operable by people with no vision (who cannot use devices such as mice that require eye-hand coordination) as well as by people who must use alternate keyboards or input devices that act as keyboard emulators. Keyboard emulators include speech input software, sip-and-puff software, on-screen keyboards, scanning software and a variety of assistive technologies and alternate keyboards. Individuals with low vision also may have trouble tracking a pointer and find the use of software much easier (or only possible) if they can control it from the keyboard.
